recipes/busco/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - findutils
     - blast >=2.2,<2.4
     - hmmer >=3.1b2 # sepp needs this version
-    - augustus
+    - augustus 3.2.3
```

Comment:
Do you need exactly this version. I suggest to use `augustus >= 3.2.3`

Reply:
For now, yes we just want 3.2.3. There are discrepancies in the results when using 3.3.x that we are trying to get to the bottom of, so we don't want >= 3.2.3.